modules/express/package.json

```diff
@@ -73,6 +73,9 @@
     "supertest-as-promised": "https://github.com/BitGo/supertest-as-promised/archive/a7f4b612b9fa090ae33a9616c41862aec2b25c7e.tar.gz",
     "typescript": "^4.2.4"
   },
+  "overrides": {
```

Comment:
only supported since v8.3: https://github.com/npm/cli/releases/tag/v8.3.0

Reply:
We can:
* Put resolution in the root for yarn installers
* Put override in modules/express

In testing with v6.14.17. Looks to resolve correctly.

Reply:
`build-docker-express` released for customers pull from the root node_modules so that will be correct as well.

Reply:
Pending `ci` tests (for removed `yarn.lock`), we only need to update `express`.